Dockerfile

```diff
@@ -33,3 +33,5 @@ ENV PGUSER: "${PGUSER:-postgres}"
 # copy the python venv into this output image and add it's bin to the path
 COPY --from=APP /opt/swoop/db/swoop-db-venv /opt/swoop/db/swoop-db-venv
 ENV PATH=/opt/swoop/db/swoop-db-venv/bin:$PATH
+
+COPY bin/db-initialization.py home/db-initialization.py
```

Comment:
Can we change the location of this script outside the home directory? Maybe somewhere within /opt/swoop/db?